OMCompiler/Compiler/BackEnd/BackendDAEUtil.mo

```diff
@@ -8572,9 +8546,6 @@ algorithm
       enabledModules := "partlintornsystem"::enabledModules;
     end if;
 
-    // handle special flags, which disable modules
-    disabledModules := deprecatedDebugFlag(Flags.DIS_SIMP_FUN, disabledModules, "simplifyComplexFunction", "postOptModules-");
-
     if Flags.getConfigString(Flags.REMOVE_SIMPLE_EQUATIONS) == "none" or
        Flags.getConfigString(Flags.REMOVE_SIMPLE_EQUATIONS) == "fastAcausal" or
        Flags.getConfigString(Flags.REMOVE_SIMPLE_EQUATIONS) == "allAcausal" then
```

Comment:
Can we have the `in` keyword so that you can write
```suggestion
    if Flags.getConfigString(Flags.REMOVE_SIMPLE_EQUATIONS) in {"none", "fastAcausal", "allAcausal"} then
```
please?
There are probably only a handful of use cases for this tho :sweat_smile:
I think I've seen this once or twice

Reply:
`if listMember(Flags.getConfigString(Flags.REMOVE_SIMPLE_EQUATIONS), {"none", "fastAcausal", "allAcausal"}) then`

Not quite as elegant, but it works.

Reply:
I do not like these String comparisons in general... i would much rather have those options be numbered and check for integers which have enumerated names in the backend. But i think that would take too much change. It works as is and is readable :D 

For those checks i would just do what @perost suggested.

Reply:
It's best as it is now, I guess. It just looks dumb.